inlang/source-code/sdk/src/persistence/filelock/releaseLock.ts

```diff
@@ -0,0 +1,27 @@
+import { type NodeishFilesystem } from "@lix-js/fs"
+import { debugLock } from "../../loadProject.js"
```

Comment:
please fix as mentioned in https://github.com/opral/monorepo/pull/2709/#pullrequestreview-2042655176